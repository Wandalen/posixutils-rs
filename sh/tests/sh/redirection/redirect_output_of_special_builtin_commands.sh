mkdir -p tests/write_dir
cd tests/write_dir

set -o > redirect_output_of_special_builtin_commands.txt
cat redirect_output_of_special_builtin_commands.txt
rm redirect_output_of_special_builtin_commands.txt