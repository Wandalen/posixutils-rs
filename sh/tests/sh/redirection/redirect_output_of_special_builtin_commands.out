allexport off
notify    off
noclobber off
errexit   off
noglob    off
hashall   off
monitor   off
noexec    off
nounset   off
verbose   off
xtrace    off
ignoreeof off
nolog     off
vi        off
